Source/ModuleSimpleBoiloff.cs

```diff
@@ -137,7 +141,9 @@ public void DoCatchup()
         {
           if (part.vessel.missionTime > 0.0)
           {
-              if (part.RequestResource("ElectricCharge", coolingCost * TimeWarp.fixedDeltaTime) < coolingCost * TimeWarp.fixedDeltaTime)
+              vessel.GetConnectedResourceTotals(PartResourceLibrary.Instance.GetDefinition("ElectricCharge").id, out double currentEC, out double maxEC);
```

Comment:
it looks like you're calling this function a lot, it's somewhat expensive. Any chance of caching in a variable?